src/app/component/MyAccountMyWishlist/MyAccountMyWishlist.component.js

```diff
@@ -187,11 +191,13 @@ export class MyAccountMyWishlist extends PureComponent {
 
         return (
             <button
-              block="Button"
+              mix={ { block: 'MyAccountMyWishlist', elem: 'ShareWishlistButton' } }
               onClick={ shareWishlist }
               disabled={ disabled }
             >
-                { __('Share Wishlist') }
+                <Image
+                  src={ media(SHARE_WISHLIST_ICON, '', false) }
```

Comment:
I think it is more preferable to import this SVG as a resource and simply render it. Like so: https://create-react-app.dev/docs/adding-images-fonts-and-files